test/e2e/framework/statefulset_utils.go

```diff
@@ -812,6 +821,65 @@ func NewStatefulSet(name, ns, governingSvcName string, replicas int32, statefulP
 	}
 }
 
+// NewStatefulSetV1beta2 creates a new NGINX v1beta2 StatefulSet for testing. The StatefulSet is named name, is in namespace ns,
+// statefulPodsMounts are the mounts that will be backed by PVs. podsMounts are the mounts that are mounted directly
+// to the Pod. labels are the labels that will be usd for the StatefulSet selector.
+func NewStatefulSetV1beta2(name, ns, governingSvcName string, replicas int32, statefulPodMounts []v1.VolumeMount, podMounts []v1.VolumeMount, labels map[string]string) *appsV1beta2.StatefulSet {
```

Comment:
@janetkuo The best is to have the function generic (able to accept both v1beta1 and v1beta2 types), but i am not sure what is the right way to do. An idea is to return v1 type and convert it later on.